packages/flame/lib/src/components/text_box_component.dart

```diff
@@ -66,26 +66,52 @@ class TextBoxComponent<T extends TextRenderer> extends TextComponent {
     String? text,
     T? textRenderer,
     TextBoxConfig? boxConfig,
+    Anchor? align,
     double? pixelRatio,
     Vector2? position,
+    Vector2? size,
     Vector2? scale,
     double? angle,
     Anchor? anchor,
     Iterable<Component>? children,
     int? priority,
   })  : _boxConfig = boxConfig ?? TextBoxConfig(),
+        _fixedSize = size != null,
+        align = align ?? Anchor.topLeft,
         pixelRatio = pixelRatio ?? window.devicePixelRatio,
         super(
           text: text,
           textRenderer: textRenderer,
           position: position,
           scale: scale,
+          size: size,
           angle: angle,
           anchor: anchor,
           children: children,
           priority: priority,
         );
 
+  /// Alignment of the text within its bounding box.
+  ///
+  /// This property combines both the horizontal and vertical alignment. For
+  /// example, setting this property to `Align.center` will make the text
+  /// centered inside its box. Similarly, `Align.bottomRight` will render the
+  /// text that's aligned to the right and to the bottom of the box.
+  ///
+  /// Custom alignment anchors are supported too. For example, if this property
+  /// is set to `Anchor(0.1, 0)`, then the text would be positioned such that
+  /// its every line will have 10% of whitespace on the left, and 90% on the
+  /// right. You can use an `AnchorEffect` to make the text gradually transition
+  /// between different alignment values.
+  Anchor align;
```

Comment:
Will the "text alignment" always be left aligned even though `center` is sent in here? I think that could be confusing, this feels like it is more like an `anchor` and and that `align` should be text alignment?

Reply:
Aah, or maybe it aligns every row separately, that could work I guess!